ui/redux/actions/content.js

```diff
@@ -220,10 +220,23 @@ export function savePosition(uri: string, position: number) {
     const { claim_id: claimId, txid, nout } = claim;
     const outpoint = `${txid}:${nout}`;
 
-    dispatch({
-      type: ACTIONS.SET_CONTENT_POSITION,
-      data: { claimId, outpoint, position },
-    });
+    try {
+      localStorage.setItem(
+        ACTIONS.SET_CONTENT_POSITION,
+        JSON.stringify({
+          claimId,
+          outpoint,
+          position,
+        })
+      );
+
+      dispatch({
+        type: ACTIONS.SET_CONTENT_POSITION,
+        data: { claimId, outpoint, position },
+      });
```

Comment:
Should we put the existing dispatch out of the try-catch?  If localStorage fails, at least the "regular" one still works.